### ISSUE TITLE
False negative for AvoidTooShortNameRule

### ISSUE THREAD

**Issue by parhamsaremi** [fsprojects/FSharpLint#592]
### Description

The code below is a false negative for AvoidTooShortNameRule:
```
[ for c in text -> c ] |> parse List.Empty |> List.toArray
```

### Expected behavior

FSharplint should raise an error when parsing the block of code.

### Actual behavior

FSharplint doesn't raise any problems with this block of code.
